Lib/asyncio/tools.py

```diff
@@ -27,6 +27,7 @@ def _index(result):
         for tid, tname, awaited in tasks:
             id2name[tid] = tname
             for stack, parent_id in awaited:
+                stack = [elem[0] if isinstance(elem, tuple) else elem for elem in stack]
```

Comment:
@ambv This code just returns the fully qualified name. Play a bit with this and decide how we want to format this in the tool